Formula/blast.rb

```diff
@@ -17,6 +17,16 @@ class Blast < Formula
   conflicts_with "proj", :because => "both install a `libproj.a` library"
 
   def install
+    # Fix shebang lines and JSON dependency
+    inreplace "c++/src/app/blast/update_blastdb.pl", "#!/usr/bin/perl", "#!/usr/bin/env perl"
+    inreplace "c++/src/app/blast/update_blastdb.pl", "JSON", "JSON::PP"
+    inreplace "c++/src/app/blast/update_blastdb.pl", "from_json", "decode_json"
```

Comment:
What's the error without these two lines? I'm inclined to remove them.

Reply:
Without lines 22 and 23, update_blastdb.pl depends on JSON, which isn't a core perl module. JSON::PP has been a core perl module since perl 5.14 and thus is more widely available.

Reply:
Works for me. Thanks for the fix, Christiam. Could you please add this explanation as a Ruby comment?

Reply:
Sure, this is done.